sarazaToussaint/Way_Farer
# #167255412 ft(view bookings):Styling user view bookings [Start #167255412]

#### What does this PR do?
  Have user view bookings UI
#### Description of Task to be completed?
     Have the task user bookings management
#### How should this be manually tested?
         User should view all his/bookings so that he/she can manage them it means delete booking
#### Any background context you want to provide?
        Design of UI with no front end framework
#### What are the relevant pivotal tracker stories?
#167255412
